fuse/context.py

```diff
@@ -231,106 +241,75 @@ def full_chain_context(
-        scale_top_ = s2_mean_area_fraction_top / orig_aft_
-        scale_bot_ = (1 - s2_mean_area_fraction_top) / (1 - orig_aft_)
-        s2map.data["map"][..., :n_top_pmts] *= scale_top_
-        s2map.data["map"][..., n_top_pmts:n_tpc_pmts] *= scale_bot_
-        s2_pattern_map.__init__(s2map.data)
-    return s2_pattern_map
-
-
-# Probably not needed!
-@URLConfig.register("simple_load")
-def load(data):
-    """Some Documentation."""
-    return data
-
-
-@URLConfig.register("simulation_config")
-def from_config(config_name, key):
-    """Return a value from a json config file."""
-    config = straxen.get_resource(config_name, fmt="json")
-    return config[key]
-
-
-class DummyMap:
-    """Return constant results with length equal to that of the input and
-    second dimensions (constand correction) user-defined."""
-
-    def __init__(self, const, shape=()):
-        self.const = float(const)
-        self.shape = shape
-
-    def __call__(self, x, **kwargs):
-        shape = [len(x)] + list(self.shape)
-        return np.ones(shape) * self.const
-
-    def reduce_last_dim(self):
-        assert len(self.shape) >= 1, "Need at least 1 dim to reduce further"
-        const = self.const * self.shape[-1]
-        shape = list(self.shape)
-        shape[-1] = 1
-
-        return DummyMap(const, shape)
-
-
-@URLConfig.register("constant_dummy_map")
-def get_dummy(const, shape=()):
-    """Make an Dummy Map."""
-    itp_map = DummyMap(const, shape)
-    return itp_map
+    """Function to create a fuse full chain simulation context with the proper
+    settings for XENONnT simulations.
+
+    It takes the general full_chain_context and sets the proper
+    corrections and configuration files for XENONnT.
+    """
+
+    import fuse
+
+    # Get the simulation config file
+    if "/" in simulation_config:
+        # Local path to the simulation config file
+        simulation_config_file = simulation_config
+    else:
+        # Get the simulation config file from private_nt_aux_files
+        simulation_config_file = "fuse_config_nt_{:s}.json".format(simulation_config)
+
+    # Get the corrections_run_id from argument or from config file
+    corrections_run_id = corrections_run_id or fuse.from_config(
+        simulation_config_file, "default_corrections_run_id"
+    )
+    log.info(f"Using corrections_run_id: {corrections_run_id}")
+
+    # Get the fdc_map_mc from argument or from config file
+    fdc_map_mc = fdc_map_mc or fuse.from_config(simulation_config_file, "fdc_map_mc")
+
+    # Get clustering method
+    # if it is specified as an argument, use that
+    # if it is not specified, try to get it from the config file
+    # if it is not in the config file, use dbscan
+    if clustering_method is None:
+        try:
+            clustering_method = fuse.from_config(simulation_config_file, "clustering_method")
+        except ValueError:
+            clustering_method = "dbscan"
```

Comment:
I think it would be nice if we print a little info to the log which clustering method was used. Just to make sure the user is aware what was picked in the end.